tripy/tests/spec_verification/op_doc_guide.md

```diff
@@ -0,0 +1,15 @@
+# Introduction
+
+Spec verification is designed to ensure that the datatypes documented in the operator documentation are accurate.
+
+# How to Verify an Operation
+
+To run the verification program on an operation, add the decorator `@constraints.dtype_info` to the operation. The inputs to the decorator will help the verifier determine the constraints on the inputs and the datatypes to verify.
+
+There are four optional inputs for the decorator `dtype_variables`, `dtype_constraints`, `param_type_specification`, and `function_name`. To learn more about how these work go to `tripy/constraints.py`
```

Comment:
I wouldn't mention the specifics of the interface at all. We might change this in the future and we don't want to have to update multiple different places when we do.